tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
Some debugs:

```
TASK [debops.owncloud : debug] *************************************************
ok: [cloud] => {
    "owncloud__occ_item": {
        "command": "config:list", 
        "get_output": true
    }
}

TASK [debops.owncloud : debug] *************************************************
ok: [cloud] => {
    "owncloud__user": "www-data"
}

TASK [debops.owncloud : debug] *************************************************
ok: [cloud] => {
    "owncloud__occ_run": {
        "changed": false, 
        "cmd": [
            "php", 
            "--file", 
            "/var/www/owncloud/occ", 
            "config:list", 
            "--output=json_pretty"
        ], 
        "delta": "0:00:00.109205", 
        "end": "2016-09-15 10:06:53.778113", 
        "failed": false, 
        "failed_when_result": false, 
        "rc": 0, 
        "start": "2016-09-15 10:06:53.668908", 
        "stderr": "", 
        "stdout": "Console has to be executed with the user that owns the file config/config.php\nCurrent user: root\nOwner of config.php: www-data\nTry adding 'sudo -u www-data ' to the beginning of the command (without the single quotes)", 
        "stdout_lines": [
            "Console has to be executed with the user that owns the file config/config.php", 
            "Current user: root", 
            "Owner of config.php: www-data", 
            "Try adding 'sudo -u www-data ' to the beginning of the command (without the single quotes)"
        ], 
        "warnings": []
    }
}
```

I use ansible 2.1.1.0.

ansible.cfg:
```

# Ansible configuration file generated by DebOps, all changes will be lost.
# You can manipulate the contents of this file via `.debops.cfg`.

[ssh_connection]

[paramiko]

[defaults]
...
remote_user = root
...
```

Also i use two vars that can be the problem:

```
# Default, access with root without sudo
ansible_ssh_user: root
ansible_become: false
```